CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Array manipulation and mathematical functions added:
   - `squeeze()`
   - `reshape()`
+  - `flatten() / ravel()`
+- Added `array_utils.h` to contain shared functions between float and fixed arrays.
```

Comment:
In general, these things should probably not be in the change log (as it is more an internal issue). But fine for now although it may be removed later (and there is no need to add similar notes later, just "user facing changes", sort of).